Doc/library/dis.rst

```diff
@@ -34,7 +34,9 @@ interpreter.
       Some instructions are accompanied by one or more inline cache entries,
       which take the form of :opcode:`CACHE` instructions. These instructions
       are hidden by default, but can be shown by passing ``show_caches=True`` to
-      any :mod:`dis` utility.
+      any :mod:`dis` utility. Furthermore, the interpreter now adapts the
+      bytecode to specialize it for the types of objects it encounters. The
+      adapted bytecode can be shown by passing ``adaptive=True``.
```

Comment:
I sort of like this better, since the mechanism is more general than just type information:

```suggestion
      bytecode to specialize it for different runtime conditions. The
      adaptive bytecode can be shown by passing ``adaptive=True``.
```

Reply:
Good point, applying this change